mk/aosp_optee.mk

```diff
@@ -115,6 +115,10 @@ LOCAL_MODULE_TAGS := optional
 LOCAL_REQUIRED_MODULES := $(local_module_deps)
 
 TA_TMP_DIR := $(subst /,_,$(LOCAL_PATH))
+ifneq ($(filter-out $(OPTEE_TA_TARGETS),$(local_optee_ta_target)),)
+TA_DEV_KIT_DIR := $(TA_DEV_KIT_DIR:$(OPTEE_TA_TARGETS)=$(local_optee_ta_target))
```

Comment:
Why not assign the desired value the first time around line 58?
What is `$(TA_DEV_KIT_DIR:$(OPTEE_TA_TARGETS)=$(local_optee_ta_target))` supposed to do?

Reply:
Indeed, it is possible to move this to line 58.
We do the substitution to replace the default configuration set by `$(OPTEE_TA_TARGETS)` to `$(local_optee_ta_target)` specified by module Android.mk. I think we can move this to line 58 like,
```
ifneq ($(local_optee_ta_target),)
TA_DEV_KIT_DIR := $(ABS_OPTEE_OS_OUT_DIR)/export-$(local_optee_ta_target)
else
TA_DEV_KIT_DIR := $(ABS_OPTEE_OS_OUT_DIR)/export-${OPTEE_TA_TARGETS}
endif
```

Reply:
Sounds good